Fancy Hats/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <title>
+            Fancy Hats
+        </title>
+        <script src="https://kit.fontawesome.com/d95ab1ac46.js"></script>
+        <link rel="stylesheet" href="/Fancy Hats/assets/css/styles.css">
+    </head>
+    <body>
+        <!-- START : WHOLE CONTAINER -->
+        <section class="contentContainer">
```

Comment:
use kebab-case for class names across the app

Reply:
All class names are now converted to kebab-case